src/core.c/Numeric.pm6

```diff
@@ -72,6 +72,12 @@ proto sub sign($, *%) is pure {*}
 multi sub sign(Numeric \x) { x.sign }
 multi sub sign(Cool \x)    { x.Numeric.sign }
 
+proto sub modf($, $?, *%) is pure {*}
+multi sub modf(Numeric $x --> List)                  { $x.modf                  }
```

Comment:
Problably want to add a type smiley on all types here.  Can't take a `modf` of a type object, can you?

Reply:
> Return two smileys in a List?

Reply:
@lizmat what about "use of undefined in numeric context" warning?

Reply:
Ah, I see.  Hmmm....  I think the reason the multis for math ops are set up this way, is because they precede the existence of type smileys.

OK, skip the type smileys for now, that is a more general issue :-)